src/components/Agreement/AgreementBindingActions.js

```diff
@@ -0,0 +1,177 @@
+import React from 'react'
+import styled from 'styled-components'
+import { PropTypes } from 'prop-types'
+import {
+  Accordion,
+  AppBadge,
+  Box,
+  Help,
+  TokenAmount,
+  useTheme,
+  GU,
+} from '@aragon/ui'
+import InfoField from './../InfoField'
+
+const AgreementBindingActions = React.memo(({ apps }) => {
+  const items = apps.map(
+    ({ actionCollateral, appName, challengeCollateral, settlementPeriod }) => [
+      <div
+        key="badge"
+        css={`
+          display: flex;
+          align-items: center;
+          margin-left: ${-1 * GU}px;
```

Comment:
Visual adjustment to make this site nicer against the expansion button (I think it looks tighter)